vortex/core/helpers.py

```diff
@@ -63,3 +64,13 @@ def job_args_match(job, app, args):
         except KeyError:
             matched = False
     return matched
+
+
+def concurrent_job_count_for_tool(app, tool, user=None):  # requires galaxy version >= 21.09
+    tool_id_regex = '/'.join(tool.id.split('/')[:-1]) + '/.*' if '/' in tool.id else tool.id
```

Comment:
@cat-bro Just wondering why this is needed?

Reply:
A tool id is version specific.  The reason behind turning fastqc/0.1.0+galaxy1 into fastqc/.* is so that it can match all versions when filtering, and fastqc/0.1.0+galaxy2 is considered the same tool.

Reply:
```suggestion
    # Match all tools, regardless of version. For example, a tool id such as "fastqc/0.1.0+galaxy1" is turned into "fastqc/.*"
    tool_id_regex = '/'.join(tool.id.split('/')[:-1]) + '/.*' if '/' in tool.id else tool.id
```